src/main/java/frc/robot/util/gyro/Pigeon.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.util.gyro;
+
+import com.ctre.phoenix.sensors.Pigeon2;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+/**
+ * Encapsulates pigeon gyroscope.
+ */
+public class Pigeon extends Gyro {
```

Comment:
do we want a gyro interface so we can easily switch between pigeon and navx if needed

Reply:
Does the Gyro abc not provide that, or would it be better to make it an interface

Reply:
it's already been decided that the bots are going to use the pigeon right? and like i brought up in another comment, we're using the pigeon class directly, so like, what's the point of the interface?